django_summernote/templates/django_summernote/widget_common.html

```diff
@@ -73,7 +72,7 @@
                     onBlur: function() {
                         origin.value = $sn.summernote('code');
                     },
-                    {% if not disable_upload %}
+                    {% if not config.disable_attachment %}
```

Comment:
Not sure if you were done your changes, but I wanted to mention again that like the `config.lazy` option, `config.disable_attachment` here won't be accurate in an iframe scenario as per my comment at https://github.com/summernote/django-summernote/pull/282#discussion_r184469183

Reply:
@lqez sorry for the late comment - but I think this is still a problem (and I notice it's still in `master` at https://github.com/summernote/django-summernote/blob/master/django_summernote/templates/django_summernote/widget_common.html#L75)